docs/src/docs/reports/failure-aspects.adoc

```diff
@@ -0,0 +1,10 @@
+= Failure Aspects View
+
+In case of many failures it is important to be able to find out the most severe failure reasons (aspects) that caused the most failed tests cases.
+
+For this the Failure Aspect View was designed.
+
+image::report-ng-11.png[align="center", alt="Failure Aspect View"]
+
+It comes with a filter as well and shows the most severe problems on the top and the number of tests that they caused to fail.
```

Comment:
Is most severe meant in respect to the number of occurences of a failure or its severity i.e. would a test with failure corridor high be listed above a failure aspect that affects several tests?

Reply:
The top list only depends on the number of occurences.

Reply:
Changed text